src/UsgsAstroFrameSensorModel.cpp

```diff
@@ -32,11 +32,21 @@ const std::string UsgsAstroFrameSensorModel::m_parameterName[] = {
     "v3"                      // 6
 };
 
+/**
+ * @description Initializes the sensor model with default values and logs the 
```

Comment:
These @description's are not rendering correctly, something about the doxygen configuration? 
![Screenshot 2024-04-12 at 2 38 01 PM](https://github.com/DOI-USGS/usgscsm/assets/13245976/34b58c9b-6ae5-44fa-8852-e4d7b25ba0de)

Reply:
should be addressed by changing the @description tag to a @brief tag on this constructor method in my latest commit.